src/axom/mint/examples/mint_heat_equation.cpp

```diff
@@ -366,7 +366,7 @@ const std::string help_string =
 /*!
  * \brief A structure that holds the command line arguments.
  */
-typedef struct
+using Arguments = struct
```

Comment:
👍

Reply:
Stylistically, why have using for this one? Is "struct Arguments {...};" antiquated?

Reply:
It's more modern for aliasing types, sure. I have not seen code ever that does "using Foo = class {...};". For C++, I still regard struct as a class with all public members so I'd still be inclined to just write "struct Foo {...};".

Reply:
I see your point and I agree.

Reply:
This one looked a bit odd to me too and I agree w/ @rhornung67 that's it's likely due to the mixing of C and C++ idioms in this file.

The change is from the `modernize-use-using` rule, and I didn't feel strongly enough about it to prevent the automatic change. If you'd prefer skipping this change, we can wrap these lines w/ 
```cpp
// NOLINTBEGIN
<code>
// NOLINTEND
```

Reply:
Done.